cmd/containerd-stargz-grpc/main.go

```diff
@@ -43,18 +45,21 @@ import (
 )
 
 const (
-	defaultAddress    = "/run/containerd-stargz-grpc/containerd-stargz-grpc.sock"
-	defaultConfigPath = "/etc/containerd-stargz-grpc/config.toml"
-	defaultLogLevel   = logrus.InfoLevel
-	defaultRootDir    = "/var/lib/containerd-stargz-grpc"
+	defaultAddress         = "/run/containerd-stargz-grpc/containerd-stargz-grpc.sock"
+	defaultConfigPath      = "/etc/containerd-stargz-grpc/config.toml"
+	defaultLogLevel        = logrus.InfoLevel
+	defaultRootDir         = "/var/lib/containerd-stargz-grpc"
+	defaultFuseManagerAddr = "unix:///run/containerd-stargz-grpc/fuse-manager.sock"
 )
 
 var (
-	address      = flag.String("address", defaultAddress, "address for the snapshotter's GRPC server")
-	configPath   = flag.String("config", defaultConfigPath, "path to the configuration file")
-	logLevel     = flag.String("log-level", defaultLogLevel.String(), "set the logging level [trace, debug, info, warn, error, fatal, panic]")
-	rootDir      = flag.String("root", defaultRootDir, "path to the root directory for this snapshotter")
-	printVersion = flag.Bool("version", false, "print the version")
+	address           = flag.String("address", defaultAddress, "address for the snapshotter's GRPC server")
+	configPath        = flag.String("config", defaultConfigPath, "path to the configuration file")
+	logLevel          = flag.String("log-level", defaultLogLevel.String(), "set the logging level [trace, debug, info, warn, error, fatal, panic]")
+	rootDir           = flag.String("root", defaultRootDir, "path to the root directory for this snapshotter")
+	fuseManagerAddr   = flag.String("fusemanager-address", defaultFuseManagerAddr, "address for the fuse manager's GRPC server")
+	enableFuseManager = flag.Bool("enable-fusemanager", false, "whether enable fuse manager")
```

Comment:
This is not needed. fusemanager-mode should be automatically enabled when fusemanager-address option is provided.